src/EventStore.Core/Cluster/EventStoreClusterClient.cs

```diff
@@ -15,10 +17,31 @@ public partial class EventStoreClusterClient : IDisposable {
 		private readonly IPublisher _bus;
 		internal bool Disposed { get; private set; }
 
-		public EventStoreClusterClient(Uri address, IPublisher bus,
+		public EventStoreClusterClient(Uri address, IPublisher bus, string tlsTargetHost = null,
 			Func<HttpMessageHandler> httpMessageHandlerFactory = null) {
+			if(tlsTargetHost == null && address.Scheme == Uri.UriSchemeHttps)
+				throw new Exception("Address scheme is https but TLS target host not specified.");
+
+			HttpMessageHandler httpMessageHandler = httpMessageHandlerFactory?.Invoke() ?? new HttpClientHandler {
+				ServerCertificateCustomValidationCallback = (message, certificate, chain, errors) => {
+					bool isValid = true;
+					chain.ChainPolicy.RevocationMode = X509RevocationMode.NoCheck;
```

Comment:
We should consider how better to support this such that revocation checks can be performed.

Reply:
Yes I suggest that we can let the user decide with a configurable switch due to performance considerations. I can create an issue for it.